src/ai/backend/manager/api/resource.py

```diff
@@ -748,6 +856,7 @@ def create_app(default_cors_options: CORSOptions) -> Tuple[web.Application, Iter
     add_route = app.router.add_route
     cors.add(add_route('GET',  '/presets', list_presets))
     cors.add(add_route('POST', '/check-presets', check_presets))
+    cors.add(add_route('GET', '/get-resources', get_available_resources))
```

Comment:
Let's rename the URL to `/available-resources` since the HTTP method name "GET" already indicates its a query API.